tests/mock/automation.py

```diff
@@ -1,15 +1,36 @@
 """Mock Abode Automation."""
 
 
-def get_response_ok(aid, name, is_active, the_type, sub_type=""):
+def get_response_ok(name, enabled, aid):
     """Return automation json."""
     return '''{
-        "id": ''' + str(int(aid)) + ''',
         "name": "''' + name + '''",
-        "actions": "a=1&m=2;",
-        "condition": "HHMM_2300_2300",
-        "is_active": "''' + str(int(is_active)) + '''",
-        "sub_type": "''' + sub_type + '''",
-        "type": "''' + the_type + '''",
-        "notify_msg": null
-  }'''
+        "enabled": "''' + str(enabled) + '''",
```

Comment:
I did try removing the double quotation marks around `''' + str(enabled) + '''` but that just results in a new error with the json.decoder:

````
Traceback (most recent call last):
  File "/Users/jamiemun/Projects/abodepy/venv/lib/python3.7/site-packages/requests_mock/mocker.py", line 230, in inner
    return func(*args, **kwargs)
  File "/Users/jamiemun/Projects/abodepy/tests/test_automation.py", line 301, in tests_automation_enable
    automation_json = json.loads(automation_text)
  File "/usr/local/Cellar/python/3.7.4_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/usr/local/Cellar/python/3.7.4_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/Cellar/python/3.7.4_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 3 column 20 (char 61)
````